rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -61,46 +61,77 @@ private BridgeSerializationUtils() {
         throw new IllegalAccessError("Utility class, do not instantiate it");
     }
 
-    public static byte[] serializeMap(SortedMap<Keccak256, BtcTransaction> map) {
-        int ntxs = map.size();
+    public static byte[] serializeRskTxWaitingForSignatures(Map.Entry<Keccak256, BtcTransaction> entry) {
```

Comment:
Maybe we should refer to this as `pegoutsWaitingForSignatures` instead of `rskTxWaitingForSignatures`?

I think that is how it's named in other places

Reply:
This method will be used for the SVP spend transaction which is not really a pegout. Maybe we can name it to `svpTxWaitingForSignatures` and the other one `pegoutsWaitingForSignatures`. wdyt?